lib/sidekiq/cli.rb

```diff
@@ -269,6 +269,19 @@ def validate!
         logger.info @parser
         die(1)
       end
+
+      errors = []
+      [:concurrency, :index, :timeout].each do |arg|
+        if options.has_key? arg
+          Integer(options[arg]) rescue errors.push("#{arg.to_s} must be an integer")
```

Comment:
Please, delete unnecessary `to_s` method. Ruby use `to_s` by default for interpolation. You can check this in irb session:
```
irb(main):001:0> "#{[1,2,3]}"
=> "[1, 2, 3]"
irb(main):002:0> "#{[1,2,3].to_s}"
=> "[1, 2, 3]"
irb(main):003:0> "#{{a: 1}}"
=> "{:a=>1}"
irb(main):004:0> "#{{a: 1}.to_s}"
=> "{:a=>1}"
```